plugin/pkg/scheduler/factory/factory.go

```diff
@@ -237,6 +245,27 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}, nil
 }
 
+func (f *ConfigFactory) getFitSchedulerPod() *api.Pod {
```

Comment:
sorry, I should have commented on this name too... I think `getNextPod()` would be a better function name.